charts/hono/templates/_helpers.tpl

```diff
@@ -461,7 +464,7 @@ quarkus:
       exporter:
         otlp:
           {{- if .dot.Values.jaegerBackendExample.enabled }}
-          endpoint: {{ printf "http://%s-jaeger-collector:4317" .dot.Release.Name | quote }}
+          endpoint: {{ printf "http://%s-jaeger-collector:4317" ( include "hono.fullname" .dot ) | quote }}
```

Comment:
Installing the chart with `--set jaegerBackendExample.enabled=true` causes an error here:
```
Error: INSTALLATION FAILED: template: 
hono/templates/hono-service-device-registry-embedded/hono-service-device-registry-secret.yaml:78:8: 
executing "hono/templates/hono-service-device-registry-embedded/hono-service-device-registry-secret.yaml" 
at <include "hono.quarkusConfig" $args>: 
error calling include: template: hono/templates/_helpers.tpl:467:66: 
executing "hono.quarkusConfig" at <include "hono.fullname" .dot>: 
error calling include: template: hono/templates/_helpers.tpl:29:31: 
executing "hono.fullname" at <.dot.Values.fullnameOverride>: 
nil pointer evaluating interface {}.Values
```

To fix this, it should be 
`include "hono.fullname" .`
above instead of 
`include "hono.fullname" .dot`.

There are 3 more places in `_helpers.tpl` where `include "hono.fullname" .dot` needs to be replaced in the same way.

Reply:
done